buyer-app/buyer-app/src/components/buyer/DataOrderCreate/DataOrderCreate.js

```diff
@@ -90,8 +93,18 @@ class DataOrderCreate extends Component {
       selectedBuyer
     } = this.state;
 
+    // Mongo notation, so far we use literal values to do 'eq' or '$in' when
+    // it's an array of possible values
+    const audienceQueryOperator = R.map(
+      aud => aud.length > 1 ? ({$in:aud}) : aud[0],
+      R.reduceBy(
+        (acc,{value})=>acc.concat(value),
+        [],
+        ({variable})=>variable,audience)
```

Comment:
```suggestion
        ({variable})=>variable,
        audience
      )
```
Maybe we can change the current format, right now looks awkward, audience is the reducer 3rd parameter, but it looks like the arrow function is broken trying to return two values separated with a comma, also the closing parenthesis looks odd...

Reply:
Or, even more idiomatic and easy to read (for the whole thing)
```
const byVariable = ({variable})=>variable
const groupFields = (acc,{value})=>acc.concat(value)

const audienceFilters = R.reduceBy(groupFields, [], byVariable, audiences)
const buildQuery = aud => aud.length > 1 ? ({$in:aud}) : aud[0]
const audienceQueryOperator = R.map(buildQuery, audienceFilters)
```

Reply:
Sounds great! I would implement the idiomatic one.